_posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc

```diff
@@ -1599,49 +1723,109 @@ Set the variables for the `account-baseline-security` module in this environment
 .infrastructure-live/security/_global/account-baseline/terragrunt.hcl
 [source,hcl]
 ----
-inputs = {
-  # Fill in your region you want to use (only used for API calls) and the ID of your security AWS account
-  aws_region     = "us-east-2"
-  aws_account_id = "<SECURITY_ACCOUNT_ID>"
+locals {
+  accounts = {
+    logs     = "409800740445"
+    security = "673123100581"
+    shared   = "384759303421"
+    dev      = "293847503945"
+    stage    = "384924092834"
+    prod     = "784260063686"
+    root     = "216044045972"
+  }
+
+  # Both buckets are created in the logs account by account-baseline-root
+  config_s3_bucket_name     = "acme-config-bucket-logs"
+  cloudtrail_s3_bucket_name = "acme-cloudtrail-logs"
+
+  # The Cloudtrail KMS Key is deployed at the logs account but it's value is an output from the root account.
+  cloudtrail_kms_key_arn = "arn:aws:kms:us-east-1:216044045972:alias/cloudtrail-acme"
+
+  # A local for convenient access to the security account root ARN.
+  security_account_root_arn = "arn:aws:iam::${local.accounts.security}:root"
+}
+input = {
+ # Fill in your region you want to use (only used for API calls) and the ID of your root AWS account
+  aws_region     = "us-east-1"
```

Comment:
NIT: This could be `local.aws_region`, I think.

Reply:
This file is outdated, I updated this earlier this morning :D

Reply:
So this is ✅

Reply:
I'll resolve this one.